Comparative Genomic Analysis of Nipah Virus Strains

This project analyzes complete cds Nipah virus genomes from India, Bangladesh & singapore, and Malaysia. It uses multiple sequence alignment and phylogenetic tree construction to identify evolutionary differences.

 Tools Used
- NCBI GenBank
- MAFFT (Multiple Sequence Alignment)
- BV-BRC tool (To obtain SNP table)
- MEGA (Phylogenetic Tree)
- Biopython (Data retrieval)

 Folders
- `original data files/`: Raw FASTA files from NCBI
- `aligned fasta/`: Aligned sequences
- `niv virus tree/`: Newick tree + PNG of phylogeny
- `scripts/`: Bioython used
- `report/`: Summary of findings
- To veiw the MSA file https://mafft.cbrc.jp/alignment/server/spool/_ho.250529210224909B158QExw6OKMJa03ezFIulsfnormal.html   


 Key Results
- Aligned 80 NiV strains
- Found 3686 total mutations & 3391 total SNP positions
- Malasyian strains showed more divergence than the Indian and Bangladeshain strains.

 Contact
If youâ€™re working on viral genomics or need freelance support, feel free to connect!




